superset-frontend/cypress-base/cypress/integration/dashboard/edit_mode.test.js

```diff
@@ -23,47 +23,54 @@ describe('Dashboard edit mode', () => {
     cy.server();
     cy.login();
     cy.visit(WORLD_HEALTH_DASHBOARD);
-    cy.get('.dashboard-header [data-test=edit-alt]').click();
+    cy.get('[data-test="dashboard-header"]')
+      .find('[data-test=edit-alt]')
+      .click();
   });
 
-  xit('remove, and add chart flow', () => {
+  it('remove, and add chart flow', () => {
```

Comment:
Happy to see this test making a comeback! I hope it works more reliably now! :)

Reply:
I added extra step with filtering first and removed .contains in step back - I had problems with this contains that cypress cannot deal with drag after this.